src/main/java/io/confluent/connect/hdfs/string/StringRecordWriterProvider.java

```diff
@@ -84,10 +85,20 @@ public void commit() {}
 
         @Override
         public void close() {
+          IOException error = null;
```

Comment:
Another valid way to address this is to use a `boolean errorOnWriterClose = false`, since we don't need to actually hold onto the exception. Both are valid, and I don't care either way. If you're in here and think it's cleaner, feel free to change it. Otherwise, the current code in the PR is entirely valid, so I'm fine with this.